display_information/display-dimensions/README.md

```diff
@@ -0,0 +1,38 @@
+# Display dimensions
+
+Display dimension features from a mobile map package.
+
+![Image showing the Display Dimensions sample](DisplayDimensions.png)
+
+## Use case
+
+Dimensions show specific lengths or distances on a map. A dimension may indicate the length of a side of a building or land parcel, or the distance between two features, such as a fire hydrant and the corner of a building.
+
+## How to use the sample
+
+When the sample loads, it will automatically display the map containing dimension features from the mobile map package. The name of the dimension layer containing the dimension features is displayed in the controls box. Control the visibility of the dimension layer with the "Dimension Layer visibility" check box, and apply a definition expression to show dimensions of over or equal to 450m in length using the "Definition Expression" checkbox.
```

Comment:
```suggestion
When the sample loads, it will automatically display the map containing dimension features from the mobile map package. The name of the dimension layer containing the dimension features is displayed in the controls box. Control the visibility of the dimension layer with the "Dimension Layer visibility" check box, and apply a definition expression to show dimensions of greater than or equal to 450m in length using the "Definition Expression" checkbox.
```